zerver/webhooks/zapier/doc.md

```diff
@@ -1,46 +1,51 @@
+# Zulip Zapier integration
+
 Zapier supports integrations with
 [hundreds of popular products](https://zapier.com/apps). Get notifications
 sent by Zapier directly in Zulip.
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
 1. Create an account on [Zapier](https://zapier.com).
 
-1. Go to [Zulip Integrations](https://zapier.com/apps/zulip/integrations).
-
-1. Scroll down to click on the app you want to connect to Zulip, or find it via
-   the search box.
-
-1. Under **Choose a Trigger**, select an event in the app you are connecting to Zulip.
+1. Go to [Zulip Integrations][1] and select the app you want to connect to Zulip.
```

Comment:
There are a lot of links in this page. Made just this one a reference link, as the rest are shorter.